src/components/Button/Button.module.css

```diff
@@ -63,25 +42,28 @@
  *    the most important action of a page.
  */
 .button--primary {
-  border-color: var(--eds-theme-color-primary-border);
-  background-color: var(--eds-theme-color-primary-background);
-  color: var(--eds-theme-color-body-foreground-inverted);
+  border-color: var(--eds-theme-color-button-primary-border);
+  background-color: var(--eds-theme-color-button-primary-background);
+  color: var(--eds-theme-color-button-primary-text);
 
   &:hover,
   &:focus {
-    background-color: var(--eds-theme-color-primary-background-hover);
-    color: var(--eds-theme-color-body-foreground-inverted);
+    border-color: var(--eds-theme-color-button-primary-border-hover);
+    background-color: var(--eds-theme-color-button-primary-background-hover);
+    color: var(--eds-theme-color-button-primary-text-hover);
   }
 
-  &:focus-visible {
-    @mixin focus;
```

Comment:
We don't need this in the individual variants because it's up in the main styles.